assets/src/components/v2/bus_shelter/sticky.tsx

```diff
@@ -0,0 +1,69 @@
+import useInterval from "Hooks/use_interval";
+import React, { useEffect, useState } from "react";
+import FlexZonePageIndicator from "../flex/page_indicator";
+
+const dummyData = [
```

Comment:
assumption that data coming from backend is already split into pages (alert count/line count determined ahead of time etc) and all passed at once rather than as individual pages. Each page would have a tag indicating what page type to be displayed and passed the relevant data.